cluster-autoscaler/cloudprovider/cloud_provider.go

```diff
@@ -150,6 +150,7 @@ const (
 	// ResourceNameCores is string name for cores. It's used by ResourceLimiter.
 	ResourceNameCores = "cpu"
 	// ResourceNameMemory is string name for memory. It's used by ResourceLimiter.
+	// Memory should always be provided in megabytes.
```

Comment:
I thought we agreed on GB.

Reply:
In the API - yes. But internally we use MB. You need to pass value in MB when creating ResourceLimiter (or GKE I added this conversion in gce_manager.go, for flags we've been doing it in main.go since they were first introduced). I just wanted to be explicit about what you need to pass in case someone implements this in some other cloudprovider in future.

Reply:
Then we need to either switch to gb or bytes. In gke api there is gb, in k8s api - bytes, and here mb. This is an accident waiting to happen :)

Reply:
That makes sense. However, this PR doesn't change what we use internally - it was MB since @aleksandra-malinowska implemented the limits for the first time. All I do is document existing state and convert in GKE cloudprovider.

Changing the logic will be a bigger change that would require more careful testing. Let's do it in a separate PR, especially since it's not a must-have for 1.0.2 (and this PR is).

Reply:
ok, please file an issue for making it more consistent.